tool/bindings_generator/translator.dart

```diff
@@ -326,13 +377,14 @@ class Translator {
   }
 
   List<code.Expression> _jsOverride(String? jsOverride,
-          [bool staticInterop = false]) =>
+          {bool staticInterop = false, bool anonymous = false}) =>
```

Comment:
nit: We should keep the naming scheme the same e.g. objectLiteral instead of anonymous